sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/conditional/access/AADConditionalAccessIT.java

```diff
@@ -33,6 +36,9 @@
     private static final String WEB_API_A_ENDPOINT = "http://localhost:8882/webapiA";
     private static final Logger LOGGER = LoggerFactory.getLogger(AADConditionalAccessIT.class);
 
+    /**
+     * Before running this test case, follow the @see<a href="https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-test-aad/README.md#what-is-aad-conditional-access-in-aad">guide</a> to start @see <a href="https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-test-aad-obo/src/test/java/com/azure/test/aad/webapi/conditional/access/webapi/a/ConditionalAccessPolicyTestWebApiA.java">WebapiA</a> and @see <a href="https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-test-aad-resource-server/src/test/java/com/azure/test/aad/conditional/access/webapi/b/ConditionalAccessPolicyTestWebApiB.java">webapiB</a> at first.
+     */
```

Comment:
![image](https://user-images.githubusercontent.com/13167207/113095433-ff52e500-9225-11eb-8f75-c27ba537fc33.png)
1. Delete first `@see`.
2. `to start` -> `to start.`. Add `.`.